ledger/redis/balance.go

```diff
@@ -187,3 +187,25 @@ func SetBalances(
 		watch...,
 	)
 }
+
+func SetBalancesCmd(ctx context.Context, tx *redisv9.Tx, reqs map[string][]*types.Balance) (
+	[]redisv9.Cmder,
+	[]string,
+	error,
```

Comment:
You always return nil, do you need error here?

Reply:
right, not needed